src/classes/Validators.js

```diff
@@ -71,7 +71,6 @@ export function OAuthScope (value) {
 /**
  * Validate whether input is a valid CMDR name
  * @param {string} value input to validate
- * @returns {boolean} whether input is a valid CMDR name
```

Comment:
Was this meant to be removed?

still returns a value, no?

Reply:
It doesn't return anything, it just throws an exception if it doesn't pass

Reply:
nevermind me. I guess I forgot how to read code. ![Pepega](https://cdn.frankerfacez.com/98/ad/98ad06b77d31d7dd67141ecf0d16c887.png)